kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/internal/codec/ResponseDecoderTest.java

```diff
@@ -126,4 +129,22 @@ private static ByteBuf toLength5ByteBuf(int i) {
         });
     }
 
+    @Test
+    void shouldNotThrowIfApiVersionsResponseUnsupported() {
```

Comment:
how about making the  testname reflect what is expected (rather than what is not)?

```suggestion
    void supportsFallbacktoApiResponseV0() {
```

Reply:
Thanks for your suggestions.